slam/config/trailbot_lds_3d.lua

```diff
@@ -32,24 +32,29 @@ options = {
 --SOME non-documented changes were made before as this was adapted from the backpack_3d.lua file
 
 TRAJECTORY_BUILDER_3D.num_accumulated_range_data = 1 --originally 1
-TRAJECTORY_BUILDER_3D.ceres_scan_matcher.translation_weight = 30 --originally 5
-TRAJECTORY_BUILDER_3D.ceres_scan_matcher.rotation_weight = 30 --originally 400
+TRAJECTORY_BUILDER_3D.ceres_scan_matcher.translation_weight = 10 --originally 5
+TRAJECTORY_BUILDER_3D.ceres_scan_matcher.rotation_weight = 1e2 --originally 400
 
 --These resolutions made the map a lot more clear 
-TRAJECTORY_BUILDER_3D.submaps.high_resolution = 0.05 --originally 0.1 
-TRAJECTORY_BUILDER_3D.submaps.low_resolution = 0.2 --originally 0.45
+-- TRAJECTORY_BUILDER_3D.submaps.high_resolution = 0.05 --originally 0.1 
+-- TRAJECTORY_BUILDER_3D.submaps.low_resolution = 0.2 --originally 0.45
+TRAJECTORY_BUILDER_3D.submaps.high_resolution = 0.1 --originally 0.1 
+TRAJECTORY_BUILDER_3D.submaps.low_resolution = 0.5 --originally 0.45
 
 MAP_BUILDER.use_trajectory_builder_3d = true --originally false
-MAP_BUILDER.num_background_threads = 4 --originally 4
+-- MAP_BUILDER.num_background_threads = 4 --originally 4
+MAP_BUILDER.num_background_threads = 16 --originally 4
 
-POSE_GRAPH.optimization_problem.huber_scale = 5e2 --originally 100 
-POSE_GRAPH.optimize_every_n_nodes = 10 --originally 90
--- POSE_GRAPH.optimize_every_n_nodes = 0 --only replace as zero when wanting to turn off global slam (good for testing different settings)
-POSE_GRAPH.constraint_builder.sampling_ratio = 0.03 --originally 0.3
+-- POSE_GRAPH.optimization_problem.huber_scale = 5e2 --originally 100 
+POSE_GRAPH.optimization_problem.huber_scale = 100
+POSE_GRAPH.optimize_every_n_nodes = 50 --originally 90
```

Comment:
Is this required? Shouldn't we want more frequent optimization like in the original config?